typhos/utils.py

```diff
@@ -1704,3 +1704,62 @@ def set_no_edit_style(object: QtWidgets.QLineEdit):
                 "QLineEdit { background: transparent }"
             )
         object.setReadOnly(True)
+
+
+def take_widget_screenshot(widget: QtWidgets.QWidget) -> Optional[QtGui.QImage]:
```

Comment:
I like how these utils aren't actually typhos-specific and may have broader use in the future